webpack/assets/javascripts/pages/statistics_page.js

```diff
@@ -0,0 +1,49 @@
+import donut from '../services/c3DonutChartService';
+
+export default {
+  generateChart: generateChart,
+  generateModalChart: generateModalChart
+};
+
+function generateChart(details, data) {
+  let dataEventHandlers = {
+    onclick: getClickHandler(details.search)
+  };
+
+  let config = donut.getDonutConfig(data, '#' + details.id + 'Chart', dataEventHandlers);
+
+  donut.generate(config);
+
+  function getClickHandler(url) {
+    if (url) {
+      // eslint-disable-next-line no-unused-vars
```

Comment:
I make it a practice to write the full signature of 3rd party callback functions whether or not I use all variable (for simple documentation purposes and to facilitate code maintenance.) In this case an unused variables error would have been thrown if I hadn't disabled it.